extensions/banner/deployment/src/main/resources/default_banner.txt

```diff
@@ -1,7 +1,8 @@
 
-           ██████╗ ██╗   ██╗ █████╗ ██████╗ ██╗  ██╗██╗   ██╗███████╗    ██╗ ██╗
-██╗██╗    ██╔═══██╗██║   ██║██╔══██╗██╔══██╗██║ ██╔╝██║   ██║██╔════╝    ╚██╗╚██╗
-╚═╝╚═╝    ██║   ██║██║   ██║███████║██████╔╝█████╔╝ ██║   ██║███████╗     ╚██╗╚██╗
-██╗██╗    ██║▄▄ ██║██║   ██║██╔══██║██╔══██╗██╔═██╗ ██║   ██║╚════██║     ██╔╝██╔╝
-╚═╝╚═╝    ╚██████╔╝╚██████╔╝██║  ██║██║  ██║██║  ██╗╚██████╔╝███████║    ██╔╝██╔╝
-           ╚══▀▀═╝  ╚═════╝ ╚═╝  ╚═╝╚═╝  ╚═╝╚═╝  ╚═╝ ╚═════╝ ╚══════╝    ╚═╝ ╚═╝           
\ No newline at end of file
+      ██████   ██    ██  •██████•  ██████•  ██   ██  ██    ██  •██████
+     ██    ██  ██    ██  ██    ██  ██   ██  ██  ██   ██    ██  ██
+     ██    ██  ██    ██  ████████  ██████   █████    ██    ██  •█████•
+     ██    ██  ██    ██  ██    ██  ██   ██  ██  ██   ██    ██       ██
+     •██████•  •██████•  ██    ██  ██   ██  ██   ██   ██████   ██████•
+        •██•
+          ██
```

Comment:
Any chance, the Q could have only one line at the bottom? The new banner is one more line. Personally, I also found the original banner too high but maybe it's just me.

Reply:
yeah - just tried it out and its looking weird to me.
 
original:
![2020-03-04_06-07-03](https://user-images.githubusercontent.com/54129/75909870-369d6980-5e4d-11ea-952d-4399d72c5740.png)

cobbs:
![2020-03-04_19-20-24](https://user-images.githubusercontent.com/54129/75909886-3ac98700-5e4d-11ea-86ba-6bc0f21ee807.png)

cobbs with one less line for the Q:
![2020-03-04_19-20-24 (1)](https://user-images.githubusercontent.com/54129/75909895-3c934a80-5e4d-11ea-821e-ee2e313fe4ae.png)

Reply:
I personally perfer the current one -feels less "alien"

Reply:
lets keep this one about the banner - the whole tag line is discussed in https://github.com/quarkusio/quarkus/pull/7478

Reply:
I cleaned up the amout of spaces so there is just one space now for indent and it feels better imo.

![2020-03-04_19-48-56](https://user-images.githubusercontent.com/54129/75912263-343d0e80-5e51-11ea-8fe5-cea9c66d73f9.png)

Reply:
better one so it doesn't get "stuck"/mangled to whatever is above it

 ![2020-03-04_19-50-38](https://user-images.githubusercontent.com/54129/75912384-6e0e1500-5e51-11ea-93f1-2cacf2240ee2.png)

Reply:
So we can't remove it from the banner because in case of Keycloak, we want a powered by Quarkus.
We can't remove the log line in case people hate banners and remove them ;)
We could remove the version but then we are back to my concern of a Quarkus banner default + powered by Quarkus which is redundant.

So either 

1. we accept the double version display
2. or we are back fighinting the isDefaultBanner problem to not display Powered By Quarkus unless someone overrides the default banner.

I have to admit, I like the Q version in the banner, it's more readable there.

Reply:
btw. i missed it but master doesn't show the tag line by default - only happens if you add your own banner

Reply:
It turns out by default (default banner) the Powered by line is simply not there. So this solved both my and guillaume's concern. That's in master and thus we are good.